src/tests/java/picard/vcf/FindMendelianViolationsTest.java

```diff
@@ -0,0 +1,148 @@
+package picard.vcf;
+
+import htsjdk.samtools.metrics.MetricsFile;
+import htsjdk.samtools.util.IOUtil;
+import htsjdk.samtools.util.TestUtil;
+import htsjdk.tribble.index.IndexFactory;
+import htsjdk.tribble.readers.AsciiLineReader;
+import htsjdk.tribble.readers.LineIteratorImpl;
+import htsjdk.variant.vcf.VCFCodec;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import picard.pedigree.Sex;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.regex.Pattern;
+
+import static picard.vcf.MendelianViolationDetector.MendelianViolation.*;
+
+/**
+ * Created by farjoun on 5/13/16.
+ */
+public class FindMendelianViolationsTest {
+
+    private static final File TEST_DATA_DIR = new File("testdata/picard/vcf");
+
+
+    @Test
+    public void testFindMedelianViolations() throws IOException {
+        final File vcfFile = new File(TEST_DATA_DIR, "NA12878.vcf");
+        final File vcfIndexFile = new File(TEST_DATA_DIR, "NA12878.vcf.idx");
+        vcfIndexFile.deleteOnExit();
```

Comment:
Also, I kept changing the vcf during testing and so having the test framework build the index for me was helpful. No need during commit, but I'll keep around as commented code to help when adding more tests